app/models/blueprint.rb

```diff
@@ -70,6 +70,10 @@ def content
     end
   end
 
+  def self.rule_attributes
+    %w(name description status version)
```

Comment:
I'm finding this pattern very strange.  It feels like strong params meets attr_protected, but is neither.  What exactly is this used for?